.ci.yaml

```diff
@@ -3114,6 +3114,7 @@ targets:
     enabled_branches:
       - flutter-\d+\.\d+-candidate\.\d+
     recipe: flutter/flutter
+    presubmit: false
```

Comment:
Can you file a CP request?

Reply:
Thanks for the reminder! Forgot that there is a process for cp now.

Reply:
https://github.com/flutter/flutter/issues/120466 for cp request